.github/workflows/pull-request-commentor.yaml

```diff
@@ -44,6 +44,13 @@ jobs:
           body: |
             /test ci/centos/k8s-e2e-external-storage/1.26
 
+      - name: Add comment to trigger external storage tests for Kubernetes 1.27
+        uses: peter-evans/create-or-update-comment@v2
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            /test ci/centos/k8s-e2e-external-storage/1.27
```

Comment:
Can we run all the tests on kubernetes 1.27 and see if deployment and E2E pass before mering the PR?

Ideally, we should have 2 PR's one for GitHub action change and another for mergify. If github action works fine, we can merge the mergigy PR as we won't be blocking other PRs from mergify if there is any issue with 1.27

Reply:
Yes, sure. I'll run all the tests.

Okay, I'll add that to another PR then.
Thanks!